### PR TITLE
Add/update wiki feature extensions

### PR DESCRIPTION
**Pull request by rajeevr2715**
**Requirements**

Add/Update wiki page for existing feature extension - Apigee and MultiAccount

### CODE REVIEW

File: feature-extensions/multiaccount/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Please enter the Jazz API Endpoint(Full URL):
+```
+
+## Example (Uninstall option)
+
+### With all the parameters as arguments
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python triggerJenkinsDeleteResources.py --jenkins-url JENKINSURL
```
Comment (devsatishm) [inline feature-extensions/multiaccount/README.md]: If this is an independent script, can we rename it as `teardown.py`? `triggerJenkinsDeleteResources` looks too detailed and can be replaced with simple name (similar to `setup`).

File: feature-extensions/multiaccount/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                                        --jenkins-username JENKINSUSER
+                                        --jenkins-password JENKINSPASSWORD --account-details all
```
Comment (devsatishm) [inline feature-extensions/multiaccount/README.md]: `--account-details all` - Would this delete all resources in 
- one secondary account (& all regions within it)? 
- one secondary account (one region within it)?
- all secondary accounts (excluding master account)?
- all secondary accounts (including master account)? - may be not!

File: feature-extensions/multiaccount/README.md (hunk 1 above)
Comment (bleggett) [reply feature-extensions/multiaccount/README.md]: it shouldn't be a separate script, like the main installer, the syntax should be `installer.py uninstall <opts>`

Single entry point for install/uninstall.

File: feature-extensions/multiaccount/README.md (hunk 2 above)
Comment (rajeevr2715) [reply feature-extensions/multiaccount/README.md]: all secondary accounts (including master account's extra regions, ie excluding default region)

File: feature-extensions/multiaccount/README.md (hunk 1 above)
Comment (rajeevr2715) [reply feature-extensions/multiaccount/README.md]: > If this is an independent script, can we rename it as `teardown.py`? `triggerJenkinsDeleteResources` looks too detailed and can be replaced with simple name (similar to `setup`).

Yes.

Comment (rajeevr2715) [reply feature-extensions/multiaccount/README.md]: > it shouldn't be a separate script, like the main installer, the syntax should be `installer.py uninstall <opts>`
> 
> Single entry point for install/uninstall.

Sure, will create a new followup PR for that.

**Pull request merged**